modules/training/ajax/getTabContent.php

```diff
@@ -139,4 +154,22 @@ function createTabHTML($contentType, $title, $tabVariables, $type)
 
     return $html;
 }
+
+/**
+ * Curl helper function.
+ *
+ * @param string $url url to get
+ *
+ * @return string
+ */
+function Curl_get($url)
+{
+    $curl = curl_init($url);
```

Comment:
It's pretty easy to install curl. But if it is too much for projects to do, then so be it. The PR will be closed.